packages/nix/mkDevShells/pf-env-bootstrap-restore.sh

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env bash
+
+# This script restores the environments EKS cluster nodes and resources to their original state
+
+set -eo pipefail
+export AWS_PAGER=""
+
+REPO_VARIABLES=$(pf-get-repo-variables)
+KUBE_DIR=$(echo "$REPO_VARIABLES" | jq -r '.kube_dir')
+
+KUBE_CONFIG_PATH="$KUBE_DIR/config"
+STATE_FILE="$KUBE_DIR/cluster_suspend_state.json"
+
+####################################################################
+# Step 0: Validation
+####################################################################
+
+if [[ ! -f $KUBE_CONFIG_PATH ]]; then
+  echo "Error: No kubeconfig file found at $KUBE_CONFIG_PATH. Create it with 'pf-update-kube' first!" >&2
+  exit 1
+fi
+
+# Get the state file
+if [[ ! -f $STATE_FILE ]]; then
+  echo "Error: No state file found at $STATE_FILE. Exiting." >&2
+  exit 1
+fi
+
+####################################################################
+# Step 1: Select the cluster
+####################################################################
+
+EKS_CLUSTER_NAME=$(kubectl config get-contexts -o name | fzf --prompt="Select a Kubernetes cluster: ")
+CONTEXT_ARGS="--context=$EKS_CLUSTER_NAME"
+
+####################################################################
+# Step 2: Confirmation
+####################################################################
+
+echo -e "You selected \n" >&2
+echo -e "Cluster: $EKS_CLUSTER_NAME\n" >&2
+read -rp "Enter name of cluster to confirm: " CONFIRM_CLUSTER
+
+if [ "$EKS_CLUSTER_NAME" != "$CONFIRM_CLUSTER" ]; then
+  echo -e "$CONFIRM_CLUSTER does not match $EKS_CLUSTER_NAME Exiting.\n" >&2
+  exit 1
+fi
+
+####################################################################
```

Comment:
This is a little confusing b/c you are mixing the block-style file with the main function style. Pick one or the other.